server/views/layouts/publicLayout.handlebars

```diff
@@ -32,6 +32,7 @@
   {{/extend}}
 
   {{{render 'head-scripts'}}}
+  {{#if klaroAnalyticsConsentCookie}}
   <script type="text/javascript">
```

Comment:
also it needs to check if  `instrumentationKey` is provided like it was in the old code `{{#if instrumentationKey}}`